app.js

```diff
@@ -1,11 +1,71 @@
+import data from './data.js';
+
 const slideMenu = () => {
   const hamburger = document.getElementById('hamburger');
   const navLinks = document.getElementById('nav-list');
+  const seeProject = document.querySelectorAll('.button');
 
   hamburger.addEventListener('click', () => {
     navLinks.classList.toggle('nav-active');
   });
 
+  [...seeProject].forEach((elem) => {
+    elem.addEventListener('click', (e) => {
+      const myData = data.find((elem) => elem.id === e.currentTarget.dataset.btnId);
+      const {
+        url, tittle, technologi1, technologi2, technologi3, text,
+      } = myData;
+
+      const section = document.createElement('section');
+      section.className = 'modal';
+      section.innerHTML = `
+      <div class="card-modal">
+          <div class="modal-body">
+          <img src="./images/close-butt.svg" class="close-modal" id="close-modal" alt="close button">
+            <img src="${url}" class="popup-img">
+          <div class= "flex-start">
+            <div class= "desktop-header">
+              <div class="desktop-h2">
+                <h2 class="modal-header">${tittle}</h2>
+              </div
+              <div class="desktop-btn">
+                <button class="button-modal-desktop" type="button">
+                  <a href="https://github.com/kakarrot92" target="_blank">See Live <img src="images/live-icon.svg" alt="live-icon for my project"></a>
+                 </button>
+                <button class="button-modal-desktop" type="button">
+                  <a href="https://github.com/kakarrot92" target="_blank">See Live <img src="images/Vector.png" alt="live-icon for my github"></a>
+                </button>
```

Comment:
- [ ] Kindly make sure that the popup follows the design for both desktop and mobile version from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, tests, margins).🙄👏
- Our design
![screenshot (8)](https://user-images.githubusercontent.com/44624138/141329993-f81a6f69-1b42-4bfe-ad42-6d45a3a06e33.png)
- Figma Design
![screenshot (9)](https://user-images.githubusercontent.com/44624138/141330191-6fb908d6-c595-4ed6-9f55-618557eedd2b.png)